### DISCUSSION TITLE
Amazfit Bip 5 Zepp OS 3.0

### DISCUSSION THREAD

**Discussion by moki76** [zepp-health/zeppos-forum#274] [Q&A] [answered]

**Comment by moki76** [comment]
Could any upgrade from 2.1 to 3.0 be possible for Amazfit Bip 5?

**Comment by htoooth** [comment, marked as answer]
@moki76  

As far as I know, there is no such plan at the moment.

**Reply by mimimix** [reply]
What about GTS 4?

**Reply by htoooth** [reply]
@Mimimix  Yes, this device is in the plan of update zepp 3.0.

**Reply by mimimix** [reply]
@htoooth Is there any progress in this area?

**Comment by Savelij-creator** [comment]
In general, Amazfit bip 5 cannot upgrade to Zepp os 3 due to hardware limitations. We'll have to wait for bip 6
